src/languageservice/utils/kubernetesResolver.ts

```diff
@@ -0,0 +1,12 @@
+export const KUBERNETES_SCHEMA_URL = 'https://raw.githubusercontent.com/garethr/kubernetes-json-schema/master/v1.14.0-standalone-strict/all.json';
```

Comment:
What is the reason to introduce this file with this PR? Feels like it could be its own PR.